Rakefile

```
@@ -3,15 +3,15 @@
 begin
   require 'rspec/core/rake_task'
   RSpec::Core::RakeTask.new(:spec)
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError
```

Do not suppress exceptions.